pages/common/echo.md

```diff
@@ -9,3 +9,11 @@
 - Print a message with environment variables:
 
 `echo {{"My path is $PATH"}}`
+
+- Print a message without the trailing newline:
+
+`echo -n {{"Hello World"}}`
+
+- Enable backslash escapes:
```

Comment:
You're right. In that case I'd suggest "Enable interpretation of backslash escapes (special characters)".